third_party/2and3/markdown/preprocessors.pyi

```diff
@@ -17,10 +17,10 @@ class HtmlBlockPreprocessor(Preprocessor):
     attrs_re: Any
     left_tag_re: Any
     markdown_in_raw: bool = ...
-    def run(self, lines): ...
+    def run(self, lines: List[str]) -> List[str]: ...
 
 class ReferencePreprocessor(Preprocessor):
```

Comment:
This class doesn't exist in the file you linked to?

Reply:
Good spot, looks like it got removed eight days ago, changes haven't been released

Reply:
Good timing. Out of scope for this PR then.

Reply:
For what it's worth it looks like this one actually did need a List